spec/classes/init_spec.rb

```diff
@@ -662,13 +693,41 @@
       it { should contain_file('ntp_conf').with_content(/^restrict -6 default kod notrap$/) }
     end
 
-    context 'to invalid type' do
-      let(:params) { { :restrict_options => [ 'i', 'dont', 'like', 'hashes' ] } }
+    [true,false,1,2.42,h = { 'a' => 'sh' }].each do |value|
+      context "to invalid type as #{value.class}" do
+        let(:params) { { :restrict_options => value } }
 
-      it do
-        expect {
-          should contain_class('ntp')
-        }.to raise_error(Puppet::Error)
+        it do
+          expect {
+            should contain_class('ntp')
+          }.to raise_error(Puppet::Error,/restrict_options must be an array \(prefered\) or a string \(will be deprecated\)/)
+        end
+      end
+    end
+  end
+
+  describe 'with restrict_localhost set' do
```

Comment:
it is already there: https://github.com/ghoneycutt/puppet-module-ntp/pull/69/files#diff-a446e4110d88d59f153d0721dad568feR351